incidents/hortense_20230306.yml

```diff
@@ -0,0 +1,23 @@
+meta_data:
+  title: Scheduled maintenance Tier-1 Hortense
+  start_date: 2023-03-06 07:00:00
+  end_date: 2023-03-10 20:00:00
```

Comment:
Shouldn't this be extended to - say - 2023-03-15 ?

Reply:
bumped to 2023-03-15 now